salt/grains/core.py

```diff
@@ -1156,6 +1156,23 @@ def os_data():
         grains.update(_windows_cpudata())
         grains.update(_windows_virtual(grains))
         grains.update(_ps(grains))
+
+        if 'Server' in grains['osrelease']:
```

Comment:
That shouldn't be a problem since this change is contained to a block that begins with `    elif salt.utils.is_windows():`.